app/models/content_item.rb

```diff
@@ -33,9 +34,6 @@ def self.create_or_replace(base_path, attributes)
         type: "conflict",
         code: "409",
         message: e.message,
-        fields: {
-          transmitted_at: [e.message],
```

Comment:
Actually after a bit further investigation, it doesn't seem that `fields` is used anywhere in particular, and it would just contain the same error message as in the `message`, so not sure if there's any value in keeping it in?